scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -12,156 +12,260 @@ output:
 
-```{r}
-# read in integrated SCE object 
+```{r read files}
+# read in the SCE object that has already been integrated
 integrated_sce <- readr::read_rds(integrated_sce_file)
 
 # read in sample metadata file 
 sample_metadata <- readr::read_tsv(sample_metadata_file)
 ```
 
-Before we set up our experiment, let's explore our dataset a little.
+## Dataset Exploration
 
-```{r}
-# First just plot by sample
-scater::plotReducedDim(integrated_sce,
-                       dimred = "fastmnn_UMAP",
-                       colour_by = "sample",
-                       point_size= 0.5,
-                       point_alpha = 0.2)
+Before we dive into differential expression, let's explore our integrated SCE object and the dataset a little more.
+
+We'll start by looking at what's inside the object.
+Here we should have both the original (uncorrected) data and the integrated (corrected) data for both the gene expression and the reduced dimensionality results.
+How are those stored in our object?
+
+```{r print sce}
+# print out entire object
+integrated_sce
+
+# look at the assay names in our object
+assayNames(integrated_sce)
+
+# look at the names of the dimensionality reduction present in the SCE object
+reducedDimNames(integrated_sce)
 ```
 
-We can also look at the sample metadata:
+When we look at the assay names we should see that there are 3 matrices, `counts`, `logcounts`, and `fastmnn_corrected`. 
+The `counts` and `logcounts` assays correspond to the uncorrected gene expression data that has been merged but NOT integrated.
+The `fastmnn_corrected` data contains the corrected gene expression data obtained from integration. 
+For this exercise we will not be using the `fastmnn_corrected` data (more on why not once we get to setting up the differential expression), but we need to be aware that it is present and be able to distinguish it from our uncorrected data. 
 
-```{r}
-sample_metadata
+In the `reducedDim` slots you should see `PCA` and `UMAP` which both correspond to the pre-integrated dimensionality reduction results.
+You should also see `fastmnn_PCA` and `fastmnn_UMAP` reduced dimensions, which correspond to the integrated results.
+
+Just like in the integration notebook, this dataset also contains the cell type annotations found in the `celltype_fine` and `celltype_broad` columns of the `colData`.
+These cell types were originally assigned in [Patel _et al._ (2022)](https://doi.org/10.1016/j.devcel.2022.04.003).
+We will use these cell type assignments to set up the DE analysis below, but they are not required.
+It's important to note that DE analysis can be applied to any sub population of interest that is shared across samples and does not need to be explicitly defined by a cell type.
+
+Because we are going to be doing DE analysis between ARMS and ERMS samples, let's start by first looking at our integrated dataset and labeling our cells by which RMS subtype they are from.
```

Comment:
```suggestion
Because we are going to be doing DE analysis between ARMS and ERMS samples, let's start by labeling cells in the integrated dataset based on their RMS subtype.
```